WrightTools/artists/_base.py

```diff
@@ -461,6 +462,73 @@ def imshow(self, *args, **kwargs):
             super().invert_yaxis()
         return out
 
+    def scatter(self, *args, **kwargs):
+        """Scatter plot a channel against two _variables_.
+        Scatter point color reflects channel values.
+        Data need not be structured.
+        If data object is not provided, scatter reverts to the [matplotlib parent method](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.scatter.html?highlight=scatter#matplotlib.axes.Axes.scatter)
+
+        Parameters
+        ----------
+        data : 2D WrightTools.data.Data object
```

Comment:
should be documented as  positional only

Reply:
ok, I think I get it.  `data` is pulled as the first element of `*args`, so it has to be positional.   @kameyer226 you had the right idea.

Is this a correction that extends to all our plot functions?

Reply:
Yeah, I think it does need to apply to other plot functions... most of them use `_parse_plot_args` which also expects data to be positional and more or less everything else to be kwargs (with additional args being passed as positional arguments to the underlying function after the x/y[/z] which are extracted from the data object

Reply:
I like the use of "parameters', 'args', and 'kwargs', but I do not know if my likes are compatible with the community here